tests/spectron/spectronConstants.js

```diff
@@ -20,5 +19,10 @@ module.exports = {
         }
     },
 
-    LOG_FILENAME_PREFIX: "logs_symphony_"
+    LOG_FILENAME_PREFIX: "logs_symphony_",
+
+    USER_A: {username:"jest01",password:"Symphony!123456",name:"jest01 jest01"},
+    USER_B: {username:"jest02",password:"Symphony!123456",name:"jest02 jest02"},
+    USER_C: {username:"jest03",password:"Symphony!123456"},
```

Comment:
I don't think you can store passwords here.

Reply:
fixed